### PR TITLE
Fix messages to show master ip rather than a pointer address.

### PR DESCRIPTION
**Pull request by rjnagal**

### CODE REVIEW

File: sources/kube.go [hunk 1]
```diff
@@ -247,7 +247,7 @@ func newKubeSource() (*KubeSource, error) {
 		Insecure: true,
 	})
 
-	glog.Infof("Using Kubernetes client %+v\n", kubeClient)
+	glog.Infof("Using Kubernetes client with master %q and version 'v1beta1'\n", *argMaster)
```
Comment (vmarmol) [inline sources/kube.go]: nit: Can we constify "v1beta1" here, below, and in the struct above so they all come from one variable?

**Comment by vmarmol** [comment]
LGTM

File: sources/kube.go (hunk 1 above)
Comment (rjnagal) [inline sources/kube.go]: done.

**Comment by vmarmol** [comment]
LGTM

**Comment by vishh** [comment]
I have fixed the integration tests and started one for this build. LGTM.

**Comment by vmarmol** [comment]
:D

**Pull request merged**